app/views/insured/families/_consumer_brokers_widget.html.erb

```diff
@@ -1,63 +1,85 @@
-<% current_broker_agency = @person&.primary_family&.current_broker_agency %>
-<% writing_agent =  @person&.primary_family&.current_broker_agency&.writing_agent %>
-<div class="panel panel-default module consumer-broker-widget">
-  <div class="panel-heading">
-    <h3 class="panel-title">
-      <strong>
-        <% if current_broker_agency && display_broker_info_for_consumer %>
-          <%= l10n("insured.your_broker_or_assister") %>
-        <% else %>
-          <%= l10n("insured.select_broker_or_assister") %>
-        <% end %>
-      </strong>
+<% if @bs4 %>
+  <% current_broker_agency = @person&.primary_family&.current_broker_agency %>
+  <% writing_agent =  @person&.primary_family&.current_broker_agency&.writing_agent %>
+  <div class="home-widget p-2">
+    <h3 class="d-flex justify-content-between">
+      <%= l10n("insured.broker_or_assister") %>
+      <%= link_to '#', data: { toggle: 'modal', target: '#read_only_modal_what_is_a_broker_or_assister' } do %>
```

Comment:
unsure about the move with this link_to; since it is not loading a new page and only opening a modal, do we need the h() surrounding it?